commands/uploader.go

```diff
@@ -373,7 +373,7 @@ func (c *uploadContext) ensureFile(smudgePath, cleanPath string) error {
 		if c.allowMissing {
 			return nil
 		}
-		return err
+		return errors.Wrapf(err, "Unable to find source for object %v (try running git lfs fetch --all)", oid)
```

Comment:
Just out of interest, @bk2204, as I haven't looked at the `errors` package before, does the `%v` output the error string (from `err`) or the `oid` string?  I was looking at [Formatted printing of errors](https://godoc.org/github.com/pkg/errors#hdr-Formatted_printing_of_errors) and trying to understand what it does.

Reply:
Oh, wait, I see -- different `errors` package; this one is local.  Got it!  :-)

Reply:
The `%v` is for `oid`.  What `errors.Wrapf(err, fmt, args...)` does is essentially `fmt.Sprintf("%s: %s", fmt.Sprintf(fmt, args...), err)`.  It formats the string and args you give it and then tacks on the message from `err` at the end.

Reply:
The output will look something like it does here: https://github.com/git-lfs/git-lfs/issues/1113#issuecomment-442245429

Reply:
Yeah, indeed, thanks -- I was just looking at the wrong package, having assumed incorrectly it wasn't locally defined.  This looks great!

Reply:
Yeah, it's confusing because there's a built-in `errors` package as well, which has none of these niceties.